app/api/v2/db.py

```diff
@@ -84,31 +84,30 @@ def create_queries():
                     dateCreated TIMESTAMP NULL DEFAULT NOW(),
                     FOREIGN KEY (officeId) REFERENCES offices (id),
                     FOREIGN KEY (partyId) REFERENCES parties (id),
-                    FOREIGN KEY (userId) REFERENCES users (id) ),
-                    ON DELETE CASCADE,
-                    ON UPDATE CASCADE;"""
+                    FOREIGN KEY (userId) REFERENCES users (id)
+                    ON DELETE CASCADE
+                    ON UPDATE CASCADE );"""
 
     votes = """CREATE TABLE IF NOT EXISTS votes(
                     id SERIAL PRIMARY KEY NOT NULL, 
-                    createdOn TIMESTAMP NULL DEFAULT NOW()
-                    createdBy VARCHAR(50) NOT NULL,
+                    createdOn TIMESTAMP NULL DEFAULT NOW(),
+                    createdBy INTEGER NOT NULL,
```

Comment:
Kindly explain why `createdBy` should be an INTEGER.

Reply:
'CreatedBy' is an integer because it references the user id. It is a Foreign Key